hardening/host-os/ansible/main.fmf

```diff
@@ -0,0 +1,110 @@
+summary: Runs ansible remediation directly on the target system
+test: python3 -m lib.runtest ./test.py
+result: custom
+environment+:
+    PYTHONPATH: ../../..
+duration: 1h
+recommend+:
+    # ansible-core is not available on RHEL-7
+    # ansible is not on RHEL-8+
+    - ansible-core
+    - ansible
+    # needed for the ini_file ansible plugin, and more
+    - rhc-worker-playbook
```

Comment:
We should agree on `recommend`/`required` indentation. `contest/main.fmf` has no indentation and so ctest proposed by me. But `hardening/*/main.fmf` has 4-space.

Reply:
I believe I was using TMT's version of indenting. When you actually export to TCMS (by letting TMT rewrite your `.fmf` file), it inserts this kind of 4-space indent into lists.

As far as I can tell, this is mostly against official YAML code style, which uses ie.
```yaml
recommend+:
- ansible-core
- ansible
- ...
```
instead of
```yaml
recommend+:
    - ansible-core
    - ansible
    - ...
```
But I believe my version is more TMT-friendly (would need to recheck).

Reply:
I'm fine with that. I'll update ctest PR with the 4-space indentation